lib/xclarity_client/client.rb

```diff
@@ -70,6 +70,10 @@ def discover_switches(opts = {})
       SwitchManagement.new(@connection).population opts
     end
 
+    def discover_storages(opts = {})
+      StorageManagement.new(@connection).population(opts)
```

Comment:
__Required:__
You can call `fetch_all` method directly, without define a new method just to call it

Reply:
Fixed it, now only fetch_all is used.